test/e2e/delete_test.go

```diff
@@ -111,6 +115,30 @@ func (tc *testContext) checkNetworksRemoved(address string) (bool, error) {
 		strings.Contains(out, "VIPEndpoint")), nil
 }
 
+// checkEnvVarsRemoved returns true if the system and service level ENV vars do not exist on the Windows instance
+func (tc *testContext) checkEnvVarsRemoved(address string) (bool, error) {
+	watchedEnvVars := []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"}
+	for _, proxyVar := range watchedEnvVars {
+		systemEnvVars, err := tc.getSystemEnvVar(address, proxyVar)
+		if err != nil {
+			return false, fmt.Errorf("error retrieving system level ENV vars: %w", err)
+		}
+		if systemEnvVars[proxyVar] != "" {
```

Comment:
```suggestion
		if _, exists := systemEnvVars[proxyVar]; exists {
```
otherwise this would erroneously pass if the var exists with value ""